src/main/java/net/neoforged/neoforge/client/model/generators/template/CustomLoaderBuilder.java

```diff
@@ -76,4 +81,8 @@ public JsonObject toJson(JsonObject json) {
 
         return json;
     }
+
+    protected static void serializeNestedTemplate(ModelTemplate template, TextureMapping textures, Consumer<JsonElement> consumer) {
+        template.create(DUMMY, textures, (id, jsonSup) -> consumer.accept(jsonSup.get()));
+    }
```

Comment:
Method was unused and thus has been removed, was told its better practice to now reference external files rather than inlining models like we used to, which this was used for in the past.

Reply:
Is this supposed to be removed? Basing this off your previous comment.

Reply:
Apex did remove it at first but it turned out that we'll need it for a subsequent PR, so it was added back and documented.